release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1515,6 +1520,16 @@ In the table below, features are marked with the following statuses:
 |-
 |TP
 
+|Driver Toolkit
+|-
+|TP
+|TP
```

Comment:
You didn't mention this in your PR comment. Can you explain?

Seems that this didn't get copied over from the 4.8 notes.

FYI @jeana-redhat

Reply:
This is noted in the Release Note tracker for 4.9. I updated the initial comment in this PR to point to the Driver Toolkit PR that added it to the 4.8 RNs.